### PR TITLE
cleanup references to obsolete virtualenv parameter/command

### PR DESCRIPTION
**Pull request by vchepkov**
#### Pull Request (PR) description
remove references to obsolete virtualenv parameter and obsolete fact

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -7,11 +7,9 @@
   "source": "https://github.com/voxpupuli/puppet-python.git",
   "project_page": "https://github.com/voxpupuli/puppet-python",
   "issues_url": "https://github.com/voxpupuli/puppet-python/issues",
-  "description": "Puppet module for Python",
```
Comment (kenyon) [inline metadata.json:10]: Why should the description be removed?

Comment (vchepkov) [reply metadata.json:10]: because it's an obsolete attribute, was deprecated long time ago

**Review by smortex — APPROVED** [review]
This makes sense I think.  The relevant code was removed in #596.

File: metadata.json (hunk 1 above)
Comment (smortex) [reply metadata.json:10]: `summary` is terser that this `description` we remove.  Maybe we can update the summary with this text or rephrase it to indicate the module capabilities?

The module description on GitHub says *Puppet module for installing and managing Python, pip, virtualenvs and Gunicorn virtual hosts.* as the README.  Maybe we can adjust this and sync the sentences?

Comment (vchepkov) [reply metadata.json:10]: I can remove the change from this PR if you like. I removed the line, because IDE told me it's wrong and I was touching the file anyway. summary suppose to be terse, imho, documentation says "one liner"

Comment (vchepkov) [reply metadata.json:10]: rebased/changed summary

Comment (smortex) [reply metadata.json:10]: That's great for me, thanks!

**Review by kenyon — APPROVED** [review]
